nevergrad/benchmark/experiments.py

```diff
@@ -30,6 +31,64 @@
 # fmt: off
 
 
+@registry.register
+def yawidebbob(seed: Optional[int] = None) -> Iterator[Experiment]:
```

Comment:
The name become more and more difficult to follow
+ maybe add a docstring for it to appear in the doc

Reply:
Thx, done.